src/meta/src/rpc/service/ddl_service.rs

```diff
@@ -673,10 +697,11 @@ where
 
     async fn resolve_private_link_info(
         &self,
+        connection_id: ConnectionId,
         properties: &mut HashMap<String, String>,
     ) -> MetaResult<()> {
         let mut broker_rewrite_map = HashMap::new();
-        const UPSTREAM_SOURCE_PRIVATE_LINK_KEY: &str = "private.links";
+        const UPSTREAM_SOURCE_PRIVATE_LINK_KEY: &str = "privatelink.targets";
```

Comment:
Why renaming here? Prefer the former one.

Reply:
@StrikeW what do you think?

Reply:
Let's keep the newer one. The previous one doesn't expose the concept of `Target`, which is the destination where RisingWave needs to route the network traffic.